sklearn/ensemble/_stacking.py

```diff
@@ -137,13 +137,20 @@ def fit(self, X, y, sample_weight=None):
 
         stack_method = [self.stack_method] * len(all_estimators)
 
-        # Fit the base estimators on the whole training data. Those
-        # base estimators will be used in transform, predict, and
-        # predict_proba. They are exposed publicly.
-        self.estimators_ = Parallel(n_jobs=self.n_jobs)(
-            delayed(_fit_single_estimator)(clone(est), X, y, sample_weight)
-            for est in all_estimators if est != 'drop'
-        )
+        if self.cv == "prefit":
+            self.estimators_ = []
+            for estimator in all_estimators:
+                check_is_fitted(estimator)
+                self.estimators_.append(deepcopy(estimator))
```

Comment:
Is the `deepcopy` required here?

Reply:
It's originally intended as a mirror of `clone` for fitted estimators. I feel this is a bit safer in case any unexpected side effects occur.

Reply:
I hope we can assume there are not side effects to calling predict?? (The first I can think of is advancing a random number generator.)

Reply:
I'm uncertain, but let's avoid the deepcopy to avoid memory issues, since the user has explicitly pre-fitted this, so perhaps it's a big model...????

Reply:
I don't have a strong take on this, and I do agree that `deepcopy` could have memory issues so I've removed it.